### PR TITLE
Update default spectrogram color scheme

### PR DESCRIPTION
**Pull request by dofuuz**
Previous PR: #830 (merged)

I made some improvement to the color scheme of spectrogram.
Details: https://github.com/dofuuz/roseus

audacity-roseus
↑ To-be
[As-is for comparison](https://github.com/audacity/audacity/assets/8174871/f3aca440-b91e-42c2-ad66-d42337c98393)

- It was perceptually smooth, but wasn't perceptually uniform. Now it is.
- The colormap is named 'Roseus'. (inspiration from Viridis)

Further readings about perceptually uniform colormaps:
https://bids.github.io/colormap/
https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html

TL;DR: The wrinkles in the color scheme were ironed out.

----

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

**Comment by Paul-Licameli** [comment]
Pleased to see you contributing again!

Who originated the name Roseus?

**Review by LWinterberg — APPROVED** [review]
Here's a pluck - image
and here the old/current pluck - 
image


The selected color is less green than before, which IMHO is a big plus, even if the rest of the changes are subtle compared to the previous version

image

Overall this has my blessing, thanks dofuuz!

**Comment by Paul-Licameli** [comment]
@LWinterberg did you notice the string change too, the name "Roseus"

**Comment by LWinterberg** [comment]
yes, and it is clarified for translators as well.

**Comment by dofuuz** [comment]
I named the colormap Roseus after popular colormap [Viridis](https://sjmgarnier.github.io/viridis/). (Viridis is default colormap of [Matplotlib](https://matplotlib.org/).)
Viridis is a Latin word means green, young. Roseus is a Latin word means pink, rosy.

If you have opinion about the name, it's last chance to change it! Actually, I'm not sure about how it feels in western culture.

**Comment by dofuuz** [comment]
Nice! I'll look forward to 3.4 release!

**Comment by LWinterberg** [comment]
> Actually, I'm not sure about how it feels in western culture.

It's hard to go wrong naming a thing something latin, unless you want to convince someone that [*disco* has something to do with learning](https://latin-dictionary.net/search/latin/disco) :)
